program-libs/macros/README.md

```
@@ -1,2 +1,16 @@
-# light-macros
-Set of proc-macros used in Light Protocol
+<!-- cargo-rdme start -->
+
+## light-macros
```

markdownlint-cli2 (0.20.0)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)